src/signal.rs

```diff
@@ -1351,3 +1351,39 @@ extern "C" fn tree_view_column_trampoline(this: *mut GtkTreeViewColumn,
         f: &Box<Fn(TreeViewColumn) + 'static>) {
     f(TreeViewColumn::wrap_pointer(this))
 }
+
+pub trait GLAreaSignals {
```

Comment:
Not really. The trait will be declared but will remain unused. I can add a cfg if you want but it's not really necessary.